src/client/Microsoft.Identity.Client/WwwAuthenticateParameters.cs

```diff
@@ -125,22 +182,49 @@ public static WwwAuthenticateParameters CreateFromWwwAuthenticateHeaderValue(str
         /// </summary>
         /// <param name="resourceUri">URI of the resource.</param>
         /// <returns>WWW-Authenticate Parameters extracted from response to the unauthenticated call.</returns>
+        [Obsolete]
```

Comment:
Add a note why obsolete and what else to use.

Reply:
Do we need to Obsolete these? Can't we add a "scheme" input param to them which defaults to bearer?

Reply:
This is what I did for the previous iteration but we had a discussion and you mentioned we should obsolete apis that are not in alignment for what we are trying to do (Which is return multiple headers). Seems like we should reconsider this approach.